fingertip/machine.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, backend_name, sealed=True, expire_in='7d'):
         self.hooks.clone.append(
             lambda to: reflink.auto(os.path.join(self.path, 'log.txt'),
                                     os.path.join(to, 'log.txt')))
+        self.exit_code = 0 # fist non-zero exit code in the pipeline
```

Comment:
why change anything in `machine.py` at all?

Reply:
Is there some better place to store it? Should I use EXIT_CODE right away?